.pre-commit-config.yaml

```diff
@@ -42,6 +42,8 @@ repos:
   - repo: local
     hooks:
       - id: mypy
+        args:
+          ["--install-types", "--non-interactive", "--ignore-missing-imports"]
```

Comment:
you should list all type-stubs libraries in the dev group of pyproject file, let's remove it from here :^)